packages/main/src/plugin/provider-registry.ts

```diff
@@ -789,7 +789,10 @@ export class ProviderRegistry {
 
   getMatchingConnectionLifecycleContext(
     internalId: string,
-    providerContainerConnectionInfo: ProviderContainerConnectionInfo | ProviderKubernetesConnectionInfo,
+    providerContainerConnectionInfo:
+      | ProviderContainerConnectionInfo
```

Comment:
```suggestion
      ProviderContainerConnectionInfo
```

should we remove this first `|`